### PR TITLE
APIGOV-23214 - document the polling agent cloudformation and vars

### PR DESCRIPTION
**Pull request by jcollins-axway**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Add documentation on the AWS Polling agent cloud formation and config variable

## Checklist for contributors

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

@vivekschauhan @sbolosan @tsjohns9 - please review at your convenience

### CODE REVIEW

**Review by tsjohns9 — APPROVED** [review]

**Review by vivekschauhan — APPROVED** [review]

**Review by lbadenhop — APPROVED** [review]

File: content/en/docs/connect_manage_environ/connect_aws_gateway/deploy-your-agents-1.md [hunk 1]
```diff
@@ -329,9 +330,9 @@ The Traceability Agent is used to filter the AWS CloudWatch logs that are relate
...
 | CENTRAL_USAGEREPORTING_PUBLISH                 | Enables/disables the sending of usage events to Amplify (default value: `true`). Takes precedence over replaced variable CENTRAL_PUBLISHUSAGE .                                                                                                                                                                                               |
-| CENTRAL_USAGEREPORTING_INTERVAL                | The frequency in which the agent reports API usage to Amplify (default value: 15m). Takes precedence over replaced variable CENTRAL_EVENTAGGREGATIONINTERVAL.                                                                                                                                                                                |
+| CENTRAL_USAGEREPORTING_INTERVAL                | The frequency in which the agent reports API usage to Amplify (default value: 15m). Takes precedence over replaced variable CENTRAL_EVENTAGGREGATIONINTERVAL.                                                                                                                                                                                 |
```
Comment (robertengaxway) [inline content/en/docs/connect_manage_environ/connect_aws_gateway/deploy-your-agents-1.md:335]: If there a minimum and maximum frequency supported by the Agent, maybe we should state it ?

File: content/en/docs/connect_manage_environ/connect_aws_gateway/deploy-your-agents-1.md [hunk 2]
```diff
@@ -329,9 +330,9 @@ The Traceability Agent is used to filter the AWS CloudWatch logs that are relate
...
 | CENTRAL_USAGEREPORTING_OFFLINE                 | Set to True to enable offline usage reporting (default value: False). Ignores CENTRAL_USAGEREPORTING_INTERVAL value as that is only for online reporting.                                                                                                                                                                                     |
 | CENTRAL_USAGEREPORTING_OFFLINESCHEDULE         | Determines how often usage numbers should be saved (default value and minimum: @hourly).                                                                                                                                                                                                                                                      |
```
Comment (robertengaxway) [inline content/en/docs/connect_manage_environ/connect_aws_gateway/deploy-your-agents-1.md:337]: If there are other values than @hourly and minimum/maximum values , we should communicate that in addition to the default value?

**Review by robertengaxway — APPROVED** [review]
I approve with some suggestions around describing minimum and maximum values when its a range define.   Also to describe the other possible values that are not the default (i.e. hourly and what else might be supported)

File: content/en/docs/connect_manage_environ/connect_aws_gateway/deploy-your-agents-1.md (hunk 1 above)
Comment (jcollins-axway) [reply content/en/docs/connect_manage_environ/connect_aws_gateway/deploy-your-agents-1.md:335]: This line hasn't changed, maybe my IDE updated the formatting. If we want to update this doc maybe a ticket should be created to address it.

**Pull request merged**